docs/analysis.md

```diff
@@ -478,38 +509,38 @@ invoke_rule(rule_obj, start_step=0, end_step=None)
 ###### Arguments
 - `rule_obj (Rule)` An instance of a subclass of `smdebug.rules.Rule` that you want to invoke.
 - `start_step (int)` Global step number to start invoking the rule from. Note that this refers to a global step. This defaults to 0.
-- `end_step (int or  None)`: Global step number to end the invocation of rule before. To clarify, end_step is an exclusive bound. The rule is invoked at `end_step`. This defaults to `None` which means run till the end of the job.
-- `raise_eval_cond (bool)` This parameter controls whether to raise the exception RuleEvaluationCondition when raised by the rule, or to catch it and log the Condition and move to the next step. Defaults to False, meaning it catches the exception and logs that the evaluation condition was met for a step and moves on to evaluate for the next step.
+- `end_step (int or  None)`: Global step number to end the invocation of rule before. To clarify, `end_step` is an exclusive bound. The rule is invoked at `end_step`. This defaults to `None` which means run till the end of the job.
+- `raise_eval_cond (bool)` This parameter controls whether to raise the exception `RuleEvaluationConditionMet` when raised by the rule, or to catch it and log the message and move to the next step. Defaults to `False`, which implies that the it catches the exception, logs that the evaluation condition was met for a step and moves on to evaluate the next step.
 
 
 ## Exceptions
-SMDebug is designed to be aware that tensors required to execute a rule may not be available at every step. Hence it raises a few exceptions which allow us to control what happens when a tensor is missing. These are available in the `smdebug.exceptions` module. You can import them as follows:
+smdebug is designed to be aware that tensors required to evaluate a rule may not be available at every step. Hence, it raises a few exceptions which allow us to control what happens when a tensor is missing. These are available in the `smdebug.exceptions` module. You can import them as follows:
 
 ```python
 from smdebug.exceptions import *
 ```
 
-Here are the exceptions and their meanings:
+Here are the exceptions (along with others) and their meaning:
 
-- `TensorUnavailableForStep` : This means that the tensor requested is not available for the step. It may be saved for a different step number. Note that when you see this exception, it means that this tensor can never become available for this step in the future.
+- `TensorUnavailableForStep` : This means that the tensor requested is not available for the step. It may have been saved or will be saved for a different step number. Note that this exception implies that the requested tensor will never become available for this step in the future.
```

Comment:
==>"will be saved for a different step number" 
Not needed. 
"saved for a different step number. " -> "saved for a different step number. You can check which steps tensor is saved for by trial.tensor('tname').steps() api [https://github.com/awslabs/sagemaker-debugger/blob/master/docs/analysis.md#steps-1]"